mbed_greentea/mbed_info.py

```diff
@@ -0,0 +1,68 @@
+"""
+mbed SDK
+Copyright (c) 2011-2015 ARM Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Author: Przemyslaw Wirkus <Przemyslaw.Wirkus@arm.com>
+"""
+
+import os
+import sys
+import json
+import platform
+import pkg_resources  # part of setuptools
+
+
+packages = ['mbed-greentea', 'mbed-host-tests', 'mbed-ls', 'pyserial']
+
+def mbed_os_info():
+    """! Returns information about host OS
+    @return Returns tuple with information about OS and host platform
+    """
+    result = (os.name,
+              platform.system(),
+              platform.release(),
+              platform.version(),
+              sys.platform)
+    return result
+
+
+def mbed_get_global_info():
+    info_dict = dict()
+
+    info_dict['os'] = dict()
+    info_dict['mbed-tools'] = dict()
```

Comment:
why `mbed-tools`, sounds like ARM own all these modules even these are just greentea dependencies. perhaps this should be just like `greentea-dependensies` or just `dependencies`